frontend/public/components/graphs/index.tsx

```diff
@@ -22,7 +22,11 @@ export const Gauge = props => <AsyncComponent loader={() => import('./graph-load
 export const QueryBrowser = props => <AsyncComponent loader={() => import('./graph-loader').then(c => c.QueryBrowser)} {...props} />;
 
 
-export type MutatorFunction = (value: string | number, options?: any) => string;
+export type HumanizeFunction = (value: string | number, initialUnit?: string, preferredUnit?: string) => {
```

Comment:
We could reuse `Humanize` type from https://github.com/openshift/console/blob/a724ff96b98dbc76bc3fa0843915129ab953bca7/frontend/public/components/utils/index.tsx#L107 .

Reply:
Oh, yeah that works too. Didn't realize that was there.